gm4_combat_expanded/data/gm4_combat_expanded/functions/armor/modifier/type/canine/wolf_checked.mcfunction

```diff
@@ -3,5 +3,5 @@
 # at world spawn
 # run from armor/active/canine/check
```

Comment:
I think this header is wrong. `armor/active` might've changed to `armor/type`? Should be a simple find and replace.